crates/aptos-protos/proto/aptos/remote_executor/v1/network_msg.proto

```diff
@@ -0,0 +1,20 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.remote_executor.v1;
+
+message NetworkMessage {
+    string sender_addr = 1;
```

Comment:
sender_addr probably should go to side channel

Reply:
ok